app/styles/_settings.scss

```diff
@@ -20,15 +20,18 @@ $duration-picker-height: 24px;
 
 // Font sizes
 $font-size-tiny: 10px;
-$font-size-x-small: 12px;
+$font-size-xxs-small: 12px;
```

Comment:
We're starting to have to much font sizes. Can't you use the closest font-size every time to achieve a similar design?